### PR TITLE
Turn on nullable by default

### PR DESCRIPTION
**Pull request by davkean**
- "Baseline" existing files by opt'd them out at the file level
- Turn on nullable by default at the project level 
- Removed (now) unneeded `#nullable enable` directives.

This "stops the bleeding" by opt'ing all new files into nullable.

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/IDependenciesChanges.cs [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 #nullable enable    
 
+#nullable disable
+
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/IDependenciesChanges.cs]: Actually is this correct?

Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/IDependenciesChanges.cs]: GitHub's choking on this PR. Will fetch locally and re-check.

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/IDependenciesChanges.cs]: That looks like the only one, the extra whitespace after enable threw off the RegEx.

**Comment by davkean** [comment]
```
E:\project-system2>findstr /sip /c:"nullable enable" *.cs

E:\project-system2>
```

**Pull request merged**